.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,3 +8,29 @@ assignees: ''
 ---
 
 Please only report technical bugs. Clarify design issues with one of our designers: Ken Brossi, Jonas Güther and Thomas Veit
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
```

Comment:
A real link to a sandbox would be great. Maybe to a sandbox that has a default config for react a our PL.

Reply:
something like this? https://codesandbox.io/s/dreamy-kilby-uc7jv

Reply:
I think the user can provide a code sandbox himself. No matter what his preferences are.

Reply:
Yes exactly. And he can provide what he want... its just a hint or help

Reply:
done